src/Octoshift/Mannequins.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using OctoshiftCLI.Models;
+
+namespace Octoshift
+{
+    public class Mannequins
+    {
+        private readonly Mannequin[] _mannequins;
+
+        public Mannequins(Mannequin[] mannequins)
+        {
+            _mannequins = mannequins;
+        }
+
+        public Mannequins(IEnumerable<Mannequin> mannequins)
```

Comment:
This class is a service and needs to be injected into other services, but this `ctor` makes it impossible to register it against the DI container. So if you really want to keep this class a service then this arg needs to be method arg rather than `ctor` arg.